src/sdk-reference/cpp/1/essentials/error-handling/snippets/error-handling.cpp

```diff
@@ -0,0 +1,10 @@
+try {
+  kuzzle->index->create("nyc-open-data");
+} catch (kuzzleio::GatewayTimeoutException e) {
+  /* Won't be called */
```

Comment:
Why mention it, if it won't be called?

Reply:
It was to give a full example on how to handle exception but it's too much babysitting I think